packages/replayio/src/utils/recordings/upload/uploadRecording.ts

```diff
@@ -117,7 +118,7 @@ export async function uploadRecording(
 
   debug("Uploaded %d bytes for recording %s", size, recording.id);
 
-  // TODO [PRO-103] Upload source-maps
+  await uploadSourceMaps(client, recording);
```

Comment:
nit: Could add some debug logs for this to potentially help ourselves later

```suggestion
  if (recording.metadata.sourcemaps.length > 0) {
    debug("Uploading %s source maps", recording.metadata.sourcemaps);

    await uploadSourceMaps(client, recording);

    debug("Uploaded %s source maps", recording.metadata.sourcemaps);
  }
```

Reply:
I already have some debug logs here within `uploadSourceMaps`. I just added an extra one here to have a clear mark when all source maps were uploaded. I could also add a "success" log for all of the singular uploads but perhaps we could reduce the noise by logging only the start and the failure. I could see myself having a different opinion on this tomorrow 😅